core/server/config/index.js

```diff
@@ -212,7 +212,8 @@ ConfigManager.prototype.set = function (config) {
         },
         deprecatedItems: ['updateCheck', 'mail.fromaddress'],
         // create a hash for cache busting assets
-        assetHash: assetHash
+        assetHash: assetHash,
+        preloadHeaders: false
```

Comment:
My apologizes. I was thinking this was similar to `_.defaults` but I clearly didn't look close enough. It's now resolved.